src/components/views/avatars/MemberAvatar.tsx

```diff
@@ -102,6 +103,7 @@ function MemberAvatar(
                       }
                     : props.onClick
             }
+            aria-label={name ? name + ". " : ""} // Full stop adds a pause between the name and the appended message.
```

Comment:
I've tried something in https://github.com/matrix-org/matrix-react-sdk/pull/12189/commits/90ea6c4cb76e0003f45d842e824ee00b656de65c

Basically the aria-label for the component itself is just the name (without dot). 
In EventTile, we override the accessible name of the DIV that contains the avatar to be name + dot.

Reply:
That won't work because giving a div both a role and a negative tab index would be yet another accessibility violation. I've added a prop to conditionally add the `.` to the aria label.
I'm not entirely convinced that this is really useful. Testing with [Orca](https://help.gnome.org/users/orca/stable/index.html.en), I don't really see a difference with/without the dot.